From 18c9fafc9437ee62ecf2572d93a63817ab05b9df Mon Sep 17 00:00:00 2001
From: Georgios Floros <gfloros@openmailbox.org>
Date: Thu, 12 Nov 2015 22:24:48 +0200
Subject: [PATCH] Fixed bug of GUI not remembering settings

Actually the command "git repo-config" is obsolete.
This patch replaces "git repo-config" with "git config".
---
 libgiggle-git/giggle-git-config-read.c  | 2 +-
 libgiggle-git/giggle-git-config-write.c | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git libgiggle-git/giggle-git-config-read.c libgiggle-git/giggle-git-config-read.c
index 7b4fdb9..7f24e1d 100644
--- libgiggle-git/giggle-git-config-read.c
+++ libgiggle-git/giggle-git-config-read.c
@@ -120,7 +120,7 @@ static gboolean
 git_config_read_get_command_line (GiggleJob  *job,
 				  gchar     **command_line)
 {
-	*command_line = g_strdup_printf (GIT_COMMAND " repo-config --list");
+	*command_line = g_strdup_printf (GIT_COMMAND " config --list");
 	return TRUE;
 }
 
diff --git libgiggle-git/giggle-git-config-write.c libgiggle-git/giggle-git-config-write.c
index e026e4f..7fd2b07 100644
--- libgiggle-git/giggle-git-config-write.c
+++ libgiggle-git/giggle-git-config-write.c
@@ -184,13 +184,13 @@ git_config_write_get_command_line (GiggleJob  *job,
 	if (priv->value) {
 		value = g_shell_quote (priv->value);
 
-		*command_line = g_strdup_printf (GIT_COMMAND " repo-config %s %s %s",
+		*command_line = g_strdup_printf (GIT_COMMAND " config %s %s %s",
 					         priv->global ? "--global" : "",
 						 priv->field, value);
 
 		g_free (value);
 	} else {
-		*command_line = g_strdup_printf (GIT_COMMAND " repo-config %s --unset %s",
+		*command_line = g_strdup_printf (GIT_COMMAND " config %s --unset %s",
 					         priv->global ? "--global" : "", priv->field);
 	}
 
